src/block/ItemFrame.php

```diff
@@ -162,6 +171,7 @@ public function onAttack(Item $item, int $face, ?Player $player = null) : bool{
 	public function onNearbyBlockChange() : void{
 		if(!$this->getSide(Facing::opposite($this->facing))->isSolid()){
 			$this->position->getWorld()->useBreakOn($this->position);
+			$this->position->getWorld()->addSound($this->position, new ItemFrameRemoveSound());
```

Comment:
`ItemFrameDestroySound` would be more consistent, though I don't see why item frames need a dedicated sound for this. We don't play specific destroy sounds or particles for nearby updates for any other block.